compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolResolver.java

```diff
@@ -472,7 +472,7 @@ private BSymbol generateStampSymbol(Name name, BType variableSourceType, BType t
         if (types.isAssignable(variableSourceType, targetType)) {
             List<BType> paramTypes = new ArrayList<>();
             paramTypes.add(variableSourceType);
-            return symTable.createOperator(name, paramTypes, targetType, InstructionCodes.STAMP);
+            return symTable.createOperator(name, paramTypes, targetType, -1);
```

Comment:
I think the convention is to use the `InstructionCodes.NOOP` instead of `-1`

Reply:
Fixed